training/src/main/java/wolox/training/models/Book.java

```diff
@@ -62,6 +64,7 @@ public String getAuthor() {
 
 
     public void setAuthor(String author) {
+        Preconditions.checkNotNull(author, "The author cannot be null");
```

Comment:
Just so you know (no need to change this), all checkNotNull return the actual object you tested, so you could write, for example in this case:
this.author = Preconditions.checkNotNull(author, "The author cannot be null");